model_versions.py

```diff
@@ -5,15 +5,6 @@
 from write_env import write_env
 import jetlag_conf
 import os, sys
```

Comment:
I think you had the right idea with "HiddenPrint", it just needs to send output to a log instead of /dev/null.